pkg/runtime/conversion_generator.go

```diff
@@ -302,11 +302,15 @@ func (g *conversionGenerator) generateConversionsForSlice(inType, outType reflec
 }
 
 func (g *conversionGenerator) generateConversionsForStruct(inType, outType reflect.Type) error {
+	errs := []string{}
```

Comment:
@liggitt - I'm afraid I don't understand. Let me describe how this currently works. What happens is:

- we first choose for what types we will generate conversions - those are the types for which this function will return nil error
- then for those that returned nil error, we generate conversions

With this change - you return an error if and only if you were returning error before. So effectively, you will not generate any additional conversion functions (no matter what types you have).